src/constants.js

```diff
@@ -21,6 +21,7 @@ export const ImageTypeLabels = {
   "edge-simplified-installer": "RHEL for Edge Simplified Installer (.iso)",
   vhd: "Microsoft Azure (.vhd)",
   vmdk: "VMWare VSphere (.vmdk)",
+  gce: "Google Cloud Platform (.gce)",
```

Comment:
The image actually has `.tar.gz` extension.

Reply:
Fixed

![Screenshot from 2023-01-30 13-22-52](https://user-images.githubusercontent.com/11712857/215475816-1fed6a67-b020-40c2-a252-4600ec4e7de9.png)